examples/demo-rollup/Cargo.toml

```diff
@@ -20,12 +20,21 @@ hex = { workspace = true }
 tendermint = "0.32"
 tokio = { version = "1", features = ["full"] }
 tracing-subscriber = "0.3.16"
+bytes = "1.1.0"
```

Comment:
The sovereign SDK re-exports the bytes crate to avoid version conflicts. Let's use that one instead!